MapboxNavigation/CarPlayManager.swift

```diff
@@ -183,9 +183,9 @@ extension CarPlayManager: CPApplicationDelegate {
         }
     }
 
-    func mapTemplate(for interfaceController: CPInterfaceController, viewController: UIViewController) -> CPMapTemplate {
+    func mapTemplate(for interfaceController: CPInterfaceController, carPlayMapViewController: CarPlayMapViewController) -> CPMapTemplate {
```

Comment:
This method is only called in one place, so `carPlayMapViewController` will only ever be the view controller that we create in `CarPlayManager.application(_:didConnectCarInterfaceController:to:)` and set as the window’s root view controller. We could simplify a lot of this class by formalizing a `carPlayMapViewController` computed property that returns `carWindow?.rootViewController as? CarPlayMapViewController`. For example, the `carPlayMapViewController` argument here becomes unnecessary.